src/System.IO.Compression.Brotli/src/Interop/Library.Linux.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+internal static partial class Library
+{
+    internal const string BrotliEncoder = "System.IO.Compression.Native.so";
```

Comment:
You do not need to have different builds for Linux vs Mac. The name can be same `System.IO.Compression.Native` on both. The runtime will append the right suffix for you. It is what we do elsewhere: https://github.com/dotnet/corefx/blob/master/src/Common/src/Interop/Unix/Interop.Libraries.cs#L15